pkg/common/stage/stage.go

```diff
@@ -202,6 +201,13 @@ func credentialsToMap(cred string) (map[string]string, error) {
 }
 
 func StageLoadCatalog(proc *process.Process, stagename string) (s StageDef, err error) {
```

Comment:
stage is a common package and it should not depend on process.

Reply:
moved pkg/common/stage to pkg/stage and  pkg/common/datalink to pkg/datalink